### ISSUE TITLE
Install fails silently when there is no `gpg` command available

### ISSUE THREAD

**Issue by dpritchett** [asdf-vm/asdf-nodejs#53]
Just spent a good while tearing my hair out here! Glad I figured it out though.

PS I love asdf and I'm glad yall made it <3

# reproduction instructions
* uninstall gpg or at least rename it so nothing shows up
* `asdf install nodejs 8.4.0`
* nothing happens
* `list nodejs`
* nothing shows up
* `brew install gpg`
* `bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring`
* `asdf install nodejs 8.4.0`
* `asdf list nodejs`
* `> 8.4.0`

**Comment by dpritchett** [comment]
This ticket is like #51 but possibly more actionable.

**Comment by joshprice** [comment]
Had this same issue. More importantly it looks like there is a download occurring and no error is apparent but no install actually takes place.

```
 $ asdf install nodejs 8.5.0                                                                                                                               % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4369    0  4369    0     0   1444      0 --:--:--  0:00:03 --:--:--  1444
$ asdf current nodejs                                                                                                                                   
version 8.5.0 is not installed for nodejs
```

**Comment by christiansakai** [comment]
Same here. 

![screen shot 2017-10-24 at 12 08 13 am](https://user-images.githubusercontent.com/7025139/31924575-87a43e26-b850-11e7-8909-2538e27b1975.png)

Also after installing GPG from brew, still does not work.

![screen shot 2017-10-24 at 12 14 52 am](https://user-images.githubusercontent.com/7025139/31924581-92dc5328-b850-11e7-845b-ad9d1dabad3e.png)

**Comment by drusepth** [comment]
I assume you're supposed to bootstrap trust / generate a GPG key or something, but there doesn't seem to be any instructions. Does anyone know?

**Comment by Stratus3D** [comment]
I believe this has been fixed by #68.
